utility/lambdas/ecs-asg-sizer/lambda_function.py

```diff
@@ -0,0 +1,115 @@
+import boto3
+import json
+import os
+
+session = boto3.session.Session()
```

Comment:
I think we've mostly been doing under_score naming for all functions, variables, etc since it's all new code and that's what the convention is for python

Reply:
Good call.  I haven't actually ever read anything about python styling!